tajo-common/src/main/java/org/apache/tajo/SessionVars.java

```diff
@@ -126,6 +126,9 @@
   NULL_CHAR(ConfVars.$TEXT_NULL, "null char of text file output", DEFAULT),
   CODEGEN(ConfVars.$CODEGEN, "Runtime code generation enabled (experiment)", DEFAULT),
 
+  TABLE_PARTITION_KEPT_EXISTING_DATA_ENABLED(ConfVars.$TABLE_PARTITION_KEPT_EXISTING_DATA_ENABLED,
```

Comment:
How about {{TABLE_PARTITION_NO_RESULT_OVERWRITE_ENABLED}} for the config name? It seems to be more intuitive for me.